compiler/imgdata2hdf5/README.md

```diff
@@ -0,0 +1,24 @@
+# imgdata2hdf5
+
+_imgdata2hdf5_ is a tool to convert raw image data (assumed to be pre-processed) to an hdf5 file.
+
+## Prerequisite
+- Raw image data pre-processed for the corresponding DNN model
+- List of data to convert (saved in the text file)
+- Python installed with _numpy_ and _h5py_ (ex: pip install numpy h5py)
```

Comment:
as install of `one-cmds` will provide a script to prepare a virtual environment, you can guide to use that.
- how-to-prepare-virtualenv.txt
- this fil will be installed in `docs` folder. refer #3236

Reply:
I've changed the message to `See docs/how-to-prepare-virtualenv.txt`